cdc/log.cc

```diff
@@ -264,6 +266,13 @@ class cdc::cdc_service::impl : service::migration_listener::empty_listener {
                     schema.ks_name(), logname));
         }
     }
+
+    static void check_that_table_has_no_counter_columns(const schema& schema) {
+        if (schema.is_counter()) {
+            throw exceptions::invalid_request_exception(format("Cannot create CDC log for a table {}.{}, because CDC for counters is not implemented",
```

Comment:
The bureaucrat in me feels the message is to "conversational". I would prefer something like "cannot create CDC log for table KS.CF: Counter support not implemented."